ffi/src/pkcs12.rs

```diff
@@ -78,10 +78,6 @@ pub mod ffi {
         pub fn hash_algorithm(&self) -> Pkcs12HashAlgorithm {
             self.0.hash_algorithm().into()
         }
-
-        pub fn iterations(&self) -> Option<Box<u32>> {
```

Comment:
Can you try with `Option<u32>` instead? I didn’t thought twice when I reviewed this, but it’s a bit strange to returned a boxed u32.

Reply:
If I remember correctly, diplomat didn't allow returning unboxed `Option<u32>`  😢

Reply:
Seems right

>diplomat-tool dotnet ./dotnet/Devolutions.Picky/Generated/ -l ./dotnet-interop-conf.toml
>A non-reference type was found inside an Option<T>: Option<u32>